test/metric/procstat.go

```diff
@@ -24,7 +24,7 @@ func (f *ProcStatMetricValueFetcher) Fetch(namespace, metricName string, stat St
 	dims = append(dims, f.getInstanceIdDimension())
 	values, err := f.fetch(namespace, metricName, dims, stat)
 	if err != nil {
-		log.Printf("Error while fetching metric value for %s: %v", metricName, err)
+		log.Printf("Error while fetching metric value for %s: %s", metricName, err.Error())
```

Comment:
For my understanding, why is this preferred?

Reply:
Its cleaner for me IMO. I don't mind using `err`.

Reply:
Can you explain why you think it's cleaner